### PR TITLE
removed check on maximum error from test_graph

### PR DESCRIPTION
**Pull request by allaffa**

### CODE REVIEW

**Comment by allaffa** [comment]
@pzhanggit @jychoi-hpc 
Review PR#143 first. PR#143 solves some issues I had with pre-commit in order to push changes. 
Then, once we agree on that and we merge, I will rebase this PR and ask you to review it.

File: tests/test_graphs.py [hunk 1]
```diff
@@ -122,18 +122,18 @@ def unittest_train_model(model_type, ci_input, use_lengths, overwrite_data=False
 
     # Set RMSE and sample MAE/max error thresholds
```
Comment (pzhanggit) [inline tests/test_graphs.py:123]: Could you update the comment also?

**Review by pzhanggit — COMMENTED** [review]
It looks good to me. One more to remove is the maximum sample error checking in https://github.com/ORNL/HydraGNN/blob/4468013857c1c1f07dfe4658b729c8e9d2533d8e/tests/test_model_loadpred.py#L40. Could you double check if we miss any other tests? Thanks.

File: tests/test_graphs.py (hunk 1 above)
Comment (allaffa) [reply tests/test_graphs.py]: Yep. Done

**Review by pzhanggit — APPROVED** [review]

**Pull request merged**